### PR TITLE
Pull request-7/26/2020

### PR DESCRIPTION
**Pull request by c0mmander8**
After you merge this pull request, replace the release notes located at https://www.a2desktop.com/release-notes with a fresh copy of the release notes located at https://github.com/a2stuff/a2d/blob/main/RELEASE-NOTES.md.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -88,14 +88,14 @@ File > Open, or double-click the file icon.
...
-    * Aux type $4000 or $4001 are packed hi-res/double-hires images. (#107)
+    * Aux type $4000 or $4001 are packed hi-res/double hi-res images. (#107)
     * 17 block files are hi-res images.
-    * 33 block files are double-hires images.
-* Font files must be type FNT ($07) and must be MGTK font resources.
+    * 33 block files are double hi-res images.
+* Font files must be type FNT ($07) and must be MGTK font resources. (This file type is officially reserved for Apple /// SOS font files, but unlikely to be confused.)
 
 To preview files of other types (e.g. view a BIN file as text) you
-can copy the appropriate preview handler, e.g. `SHOW.TEXT.FILE` from
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:97]: Please revert this. The backticks render text in `code style`, preferred for filenames.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Project page: https://github.com/a2stuff/a2d/
 Numbers like (#123) refer to items in the issue tracker.
 https://github.com/a2stuff/a2d/issues
 
-## 1.2 - alpha
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:10]: Please revert, leave the document structure alone.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -115,12 +115,12 @@ the Apple menu.
 * Empty directories can be copied/moved. (#121)
 * Ctrl+Reset quits cleanly back to ProDOS (except buggy emulators). (#141)
 * Prevent crash with more than 8 removable devices.
-* Desk Accessories:
+* Desk accessories:
   * Date: Read-only on systems with a clock. On systems without a clock, date is saved for next session. (#30, #39)
   * Calculator: don't mis-paint when moved offscreen and other fixes. (#33, #34)
   * Sort Directory: don't need to click before sorting. (#17)
-* Hardware/Emulator Specific:
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:122]: Please revert, leave this title case.

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -115,12 +115,12 @@ the Apple menu.
 * Empty directories can be copied/moved. (#121)
 * Ctrl+Reset quits cleanly back to ProDOS (except buggy emulators). (#141)
 * Prevent crash with more than 8 removable devices.
-* Desk Accessories:
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:118]: Please revert, leave this title case.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -130,13 +130,13 @@ the Apple menu.
...
+* Special > Disk Copy app is not substantially modified, and may suffer bugs fixed in DeskTop (e.g. SmartPort behavior).
+* Other issues can be found at https://github.com/a2stuff/a2d/issues.
 
 
 # 1.1
 
-Final release by VersionSoft/Apple Computer. November 26, 1986.
+Final release by VersionSoft/Apple Computer dated November 26, 1986.
 
-See Jay Edwards' [MouseDesk/DeskTop History](https://mirrors.apple2.org.za/ground.icaen.uiowa.edu/MiscInfo/Misc/mousedesk.info)
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:141]: Please revert, leave this title case.

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -115,12 +115,12 @@ the Apple menu.
 * Empty directories can be copied/moved. (#121)
 * Ctrl+Reset quits cleanly back to ProDOS (except buggy emulators). (#141)
 * Prevent crash with more than 8 removable devices.
-* Desk Accessories:
+* Desk accessories:
   * Date: Read-only on systems with a clock. On systems without a clock, date is saved for next session. (#30, #39)
   * Calculator: don't mis-paint when moved offscreen and other fixes. (#33, #34)
   * Sort Directory: don't need to click before sorting. (#17)
-* Hardware/Emulator Specific:
-  * IIc Plus: don't spin slot 5 drives constantly. (Use Special > Check Drive) (#25)
```
Comment (inexorabletash) [inline RELEASE-NOTES.md:123]: Please revert, no need to capitalize.

**Comment by c0mmander8** [comment]
As referenced in issue #246, the release notes located at https://www.a2desktop.com/release-notes have heavy formatting throughout, including other instances of filenames, while the release notes located at https://github.com/a2stuff/a2d/blob/main/RELEASE-NOTES.md are otherwise plain text.

**Comment by c0mmander8** [comment]
As referenced in issue #246, I increased the size of "1.2" to match the larger size "1.1" is already at.

**Comment by c0mmander8** [comment]
As referenced in issue #246, other mentions of "Slot" in the release notes are already capitalized.

**Comment by c0mmander8** [comment]
As referenced in issue #247, the release notes located at https://www.a2desktop.com/release-notes have heavy formatting throughout, including other instances of filenames, while the release notes located at https://github.com/a2stuff/a2d/blob/main/RELEASE-NOTES.md are otherwise plain text.

**Comment by c0mmander8** [comment]
As referenced in issue #248, I increased the size of "1.2" to match the larger size "1.1" is already at.

**Comment by c0mmander8** [comment]
As referenced in issue #249, other mentions of "Slot" in the release notes are already capitalized.

**Comment by c0mmander8** [comment]
Should I change all mentions of capital "Slot" to lowercase "slot" because it is actually lowercase in the technotes?

**Comment by inexorabletash** [comment]
Yes, aligning with tech notes makes sense - thanks for asking.

**Comment by inexorabletash** [comment]
Let me know when you're "done"

I plan to merge, then do an edit pass to apply formatting (make menu commands bold, make the issue numbers into links, etc) then remove the version on the website.

**Comment by c0mmander8** [comment]
I'm "done", at least for the next few days. Thank you for your comment.

**Pull request merged**